CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0]
```

Comment:
Nit: Patch seems more appropriate here, since we're "fixing" the problem of missing license information (where some contributions _were already licensed_). Doesn't strongly matter though.

Reply:
I'll keep that in mind to be more conservative with versioning in the future!